docs/changelog.md

```diff
@@ -12,6 +12,14 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **browser-engine-gecko**, **browser-engine-gecko-beta**, **browser-engine-gecko-nightly**
+  * Added `GeckoPort.senderUrl` which returns the associated content URL.
+
+* **feature-accounts**
+  * It should now be possible to log-in on stable, stage and china FxA servers using a WebChannel flow.
+  * `FxaWebChannelFeature` takes a `ServerConfig` as 4th parameter to ensure incoming WebChannel messages
```

Comment:
Let's mark this as a breaking change (there are examples in this files). We need to put up PRs to Fenix and RB in anticipation of this landing too, if you don't mind. Otherwise we'll break master.